marketplace_builder/views/pages/use-cases/e-commerce/constants-and-seeding-configuration-data.liquid

```diff
@@ -0,0 +1,326 @@
+<p>Only admins can see this now.</p>
+{% endraw %}
+```
+
+By including this line on top of every page in your application you ensure that constants will be available in every included partial further down the partials tree.
+
+### Step 3. Include as needed in other parts of application
+
+Including it on top of the page does not make the constants available in:
+
+- application layout
+- inside of [YAML front matter](/how-platformos-works/glossary#frontmatter-yaml) of form configurations
+- authorization policies
+- notifications
+
+Whenever you need access to constants inside any of these places, you’ll need to include the partial again.
+
+## Seeding configuration data
+
+Brand and product type models are pretty basic ones, they only contain a name of the object. In the future though we might want to extend them with descriptions, photos, icons, etc. Storing these values in a partial would quickly become very cumbersome and limit the usefulness of GraphQL searches.
+
+### Step 1. Create data set files
+
+#### marketplace_builder/views/partials/data/brand.json.liquid
+
+```json
+[
+  "adidas",
+  "adidas Originals",
+  "Airstep Socks",
+  "Alexander Del Rossa",
+  "alpine swiss",
+  "Amazon Essentials",
+  "Arctic Extreme",
+  "ARRIS",
+  "boxercraft",
+  "Calvin Klein",
+  "Carhartt",
+  "Champion",
+  "Columbia",
+  "Copper Fit",
+  "CYZ Collection",
+  "Debra Weitzner",
+  "DG Hill",
+  "Dickies",
+  "Dockers",
+  "Enerwear",
+  "FITEXTREME",
+  "fresh tees",
+  "Fruit of the Loom",
+  "Gildan",
+  "Gold Toe",
+  "Hanes",
+  "Hanes Ultimate",
+  "Happypop",
+  "HSELL",
+  "Legendary Whitetails",
+  "Lucky Brand",
+  "Marino Avenue",
+  "Mottee&Zconia",
+  "NY Threads",
+  "ororo",
+  "People Socks",
+  "PUMA",
+  "Ross Michaels",
+  "Saucony",
+  "Socksmith",
+  "Southpole",
+  "Stance",
+  "Tesla",
+  "Thermajohn",
+  "Under Armour",
+  "USBingoshop",
+  "Wantdo",
+  "WANTDO",
+  "Wrangler"
+]
+```
+
+#### marketplace_builder/views/partials/data/product_type.json.liquid
+
+```json
+["Crew Neck", "V Neck", "Henley - Y Neck", "Polo", "Scoop Neck"]
+```
+
+`.json` part of the file name is optional, it just helps identify file type at a glance.
+
+{% capture content %}
+Many editors allow you to set syntax highlighting depending on more than just one extension. For example, in VS Code you can add below entry in editor settings.
+
+`"files.associations": { "*.json.liquid": "json" }`
+
+{% endcapture %}
+{% include 'alert/tip', content: content %}
+
+### Step 2. Create processor partials
+
+Processor files are supposed to be included inside of the migration file and should allow multiple execution without creating any duplicates. This way if you need to add more customizations in the future you can create a new migration, include processor partial and do not repeat the logic behind it.
+
+#### marketplace_builder/views/partials/migrations/create_brands.liquid
+
+```liquid
+{% raw %}
+{% query_graph 'get_brands', result_name: existing %}
```

Comment:
result_name: 'existing'

Reply:
Thanks, I'll update that. @pavelloz please disregard this and ecommerce branches for now, I'll be opening new PRs for every article, including some changes Diana asked for.